### PR TITLE
Refactored the code to fit MVC pattern.

### PR DESCRIPTION
**Pull request by Zinarch**
Git lied to me, was not as bad as i thought..

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Nice work!

File: Application/src/Main.java [hunk 1]
```diff
@@ -10,15 +10,15 @@
...
 
-import net.Net;
-import visuals.DiagramView;
-import visuals.ExecutionLog;
+import controller.network.Net;
+import view.DiagramView;
+import view.ExecutionLog;
 import model.Menu;
-import visuals.handlers.Resizer;
+import view.handlers.Resizer;
```
Comment (pontuslaestadius) [inline Application/src/Main.java]: Please order the imports, like Model first, view after, controller last.
(only in this file, the other files look fine)

File: Application/src/controller/network/Decode.java [hunk 2]
```diff
@@ -144,7 +143,7 @@ private String removeCharactersFromString(String string, char... characters) {
     }
 
     /**
-     * Given a string following the network protocol, will create a new diagram with the provided classes.
+     * Given a string following the controller.network protocol, will create a new diagram with the provided classes.
```
Comment (pontuslaestadius) [inline Application/src/controller/network/Decode.java]: Was this auto replaced?

File: Application/src/controller/network/Net.java [hunk 3]
```diff
@@ -27,7 +27,7 @@ public void run() {
     }
 
     /**
-     * Only allow a single network thread and it must be run on an FXApplicationThread.
+     * Only allow a single controller.network thread and it must be run on an FXApplicationThread.
```
Comment (pontuslaestadius) [inline Application/src/controller/network/Net.java]: Auto replaced?

File: Application/src/view/visuals/Draw.java [hunk 4]
```diff
@@ -1,11 +1,16 @@
...
 
 import javafx.scene.image.Image;
 
-import visuals.handlers.Animation;
+import view.DiagramView;
+import view.handlers.Animation;
+import view.visuals.component.ActorClass;
+import view.visuals.component.Coordinates;
+import view.visuals.component.DiagramClass;
+import view.visuals.component.Message;
```
Comment (pontuslaestadius) [inline Application/src/view/visuals/Draw.java]: Just import everything from view maybe?

File: Application/src/controller/network/Decode.java (hunk 2 above)
Comment (Sebastian-Fransson) [reply Application/src/controller/network/Decode.java]: Yes ill change that back. IntelliJ did not recognize the moving around of stuff if i did not rebuild which apparently had such side effects.

File: Application/src/controller/network/Net.java (hunk 3 above)
Comment (Sebastian-Fransson) [reply Application/src/controller/network/Net.java]: Yes ill change that back. IntelliJ did not recognize the moving around of stuff if i did not rebuild which apparently had such side effects.

File: Application/src/view/visuals/Draw.java (hunk 4 above)
Comment (Sebastian-Fransson) [reply Application/src/view/visuals/Draw.java]: When i was moving stuff around it intelliJ did not recognize them if i tried to import everything through *. I can change it here on git though hoping it works?

**Review by IsabelleTornqvist — APPROVED** [review]

**Review by timjon — APPROVED** [review]

**Review by pontuslaestadius — APPROVED** [review]

**Pull request merged**